CMSgov/design-system
# [WNMGDS-853] Generate proptypes automatically

## Summary
- This PR adds `babel-plugin-typescript-to-proptypes` as a dep to the CMSDS scripts and to our babel.config to automatically generate PropTypes for components written in Typescript. 

## How to test
- Reproduce the error by running CRA example, and using the `InlineError` component. Provide some incorrect props to confirm PropTypes are not working
- Run and build this branch, and install it to the CRA example. Ensure PropTypes are working
